client/js/libs/handlebars/ircmessageparser/findNames.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+function findNames(text) {
```

Comment:
Assuming this PR gets accepted overall, these 2 files + changes to `parse` could totally use some test cases :)
Don't worry about it for now though, let's get the rest of the PR settled first.

Reply:
👍